test/OpenTelemetry.Instrumentation.Dependencies.Tests/OpenTelemetry.Instrumentation.Dependencies.Tests.csproj

```diff
@@ -20,14 +20,15 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.Data.SqlClient" Version="1.1.1" />
+    <PackageReference Include="Microsoft.Data.SqlClient" Version="1.1.1" Condition="'$(TargetFramework)' != 'net452'" />
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.5.0" />
     <PackageReference Include="Moq" Version="4.11.0" />
     <PackageReference Include="xunit" Version="2.4.1" />
     <PackageReference Include="xunit.runner.visualstudio" Version="2.4.1">
       <PrivateAssets>all</PrivateAssets>
       <IncludeAssets>runtime; build; native; contentfiles; analyzers</IncludeAssets>
     </PackageReference>
+    <PackageReference Include="Newtonsoft.Json" Version="12.0.2" Condition="'$(TargetFramework)' == 'net461' OR '$(TargetFramework)' == 'net452'" />
```

Comment:
Let's not bring in Newtonsoft.Json for net461, it is using System.Text.Json.

Reply:
Scratch that, it looks like OpenTelemetry.Instrumentation.Dependencies.Tests.csproj is using both. Fine to keep!

Reply:
Actually, that is not entirely true. It looks like OpenTelemetry.Instrumentation.Dependencies.Tests is using both. But net461 & netcoreapp3.1 targets get their Newtonsoft.Json ref through Microsoft.Data.SqlClient (Microsoft.Data.SqlClient -> Microsoft.IdentityModel.Protocols.OpenIdConnect -> Newtonsoft.Json).

You should be able to do:
`<PackageReference Include="Newtonsoft.Json" Version="12.0.2" Condition="'$(TargetFramework)' == 'net452'" />`